### DISCUSSION TITLE
Please Allow Comments to be Locked on Gists

### DISCUSSION THREAD

**Discussion by soatok** [community/community#127477] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Examples:
1. https://gist.github.com/joepie91/5a9909939e6ce7d09e29
2. https://gist.github.com/soatok/8aef6f67fec9c702f510ee24d19ef92b?permalink_comment_id=5079125#gistcomment-5079125

I have no way to lock the comments on this Gist. This is causing me much annoyance.

Please allow Gist owners to lock the comment thread.

**Comment by unusualevent** [comment]
I honestly don't see a reason for why they wouldn't be - especially when you can lock issues. This sounds important for the sanity of users.

**Comment by issyl0** [comment]
Hello from GitHub! Thanks for this feedback! I made an internal tracking issue for this feature and have notified its owning team. To set expectations, I can't guarantee an immediate fix, but please know we're listening. ❤️

**Reply by shaun3141** [reply]
Hey @issyl0 , really impressive that you're able to respond here ❤️  

Wanted to +1 that I'd love the ability to lock gists. One use-case I am doing is hosting sample code as part of the interview process on Gists. Every once in awhile, candidates are commenting on or answering in a comment on the Gist vs. the assessment, which then every other candidate can see so I have to be really fast on the deletions.

**Comment by issyl0** [comment, marked as answer]
Good news! We [just added the ability to disable comments for Gists](https://github.blog/changelog/2025-01-06-you-can-now-disable-comments-on-github-gists/). Happy new year!

**Comment by OmarrSakr** [comment]
Hi soatok,

I understand your frustration. Currently, GitHub doesn’t provide a built-in feature for locking comments on Gists, but I agree that this would be a useful addition, especially for projects that are no longer actively maintained or for content where additional comments aren't needed.

It would be great if GitHub could implement this functionality to allow Gist owners to lock the comment threads. I encourage you to submit this as a feature request directly to GitHub's support or through their official feedback channels to increase visibility on this issue.

Feel free to check if this feature is available through GitHub updates or use other methods, such as disabling discussions in repositories or using custom comments systems if applicable.

Hope this helps!
